<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>IMPaC-TB: Integrated analysis and dynamical systems modeling of experimental TB immunology data</title>
  <meta name="description" content="Impact lab guidebook" />
<<<<<<< HEAD
  <meta name="generator" content="bookdown 0.14 and GitBook 2.6.7" />
=======
  <meta name="generator" content="bookdown 0.11 and GitBook 2.6.7" />
>>>>>>> 377dd4d87328997d547aca3e4f4d2a05b33dc190

  <meta property="og:title" content="IMPaC-TB: Integrated analysis and dynamical systems modeling of experimental TB immunology data" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://lyonsm7.github.io/impactb_book/" />
  
  <meta property="og:description" content="Impact lab guidebook" />
  <meta name="github-repo" content="lyonsm7/impactb_book" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="IMPaC-TB: Integrated analysis and dynamical systems modeling of experimental TB immunology data" />
  
  <meta name="twitter:description" content="Impact lab guidebook" />
  

<meta name="author" content="Brooke Anderson PhD, Mike Lyons PhD, Amy Fox, Burton Karger" />


<meta name="date" content="2019-12-12" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  

<link rel="next" href="day-247.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />









<<<<<<< HEAD
=======
<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>
>>>>>>> 377dd4d87328997d547aca3e4f4d2a05b33dc190

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">IMPAC-TB Lab Guidebook</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="day-247.html"><a href="day-247.html"><i class="fa fa-check"></i><b>2</b> Day -247</a></li>
<li class="chapter" data-level="3" data-path="day-242.html"><a href="day-242.html"><i class="fa fa-check"></i><b>3</b> Day -242</a></li>
<li class="chapter" data-level="4" data-path="day-240.html"><a href="day-240.html"><i class="fa fa-check"></i><b>4</b> Day -240</a></li>
<li class="chapter" data-level="5" data-path="day-233.html"><a href="day-233.html"><i class="fa fa-check"></i><b>5</b> Day -233</a></li>
<li class="chapter" data-level="6" data-path="deaths.html"><a href="deaths.html"><i class="fa fa-check"></i><b>6</b> Deaths</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">IMPaC-TB: Integrated analysis and dynamical systems modeling of experimental TB immunology data</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">IMPaC-TB: Integrated analysis and dynamical systems modeling of experimental TB immunology data</h1>
<p class="author"><em>Brooke Anderson PhD, Mike Lyons PhD, Amy Fox, Burton Karger</em></p>
<p class="date"><em>2019-12-12</em></p>
</div>
<div id="intro" class="section level1">
<h1><span class="header-section-number">Chapter 1</span> Introduction</h1>
<p>The overall objective of the data analysis and mathematical modeling component for the CSU mouse immunology experimental studies is to develop an iterative framework to identify key biological components of immunity and to quantify their relationships to one another in both data-driven and mechanistic models for the purpose of evidence-based decision-making for tuberculosis (TB) vaccine development. The data sharing plan (DSP) will include the experimental data, the quantitative analysis framework, and an application programming interface (API) to the results. As detailed in our respective biosketches, we have established expertise in all critical areas of this data analysis project.</p>
<p>The host immune response to TB vaccination and infection is complex and involves interactions between large networks of molecular and cellular constituents that vary in time and location within the host. The experimental data will be generated from a wide range of measurements and across multiple scales; including measures of disease pathology, cellular and chemical measurements for cell type and cytokine concentrations, and intracellular measurements involving RNA expression and proteomics. The conceptual basis for our proposed data analysis and modeling framework is described in a summary of the recent National Institute of Allergy and Infectious Diseases (NIAID) workshop, ‘Complex Systems Science, Modeling and Immunity’ [1]. The major components of this framework are illustrated in Figure 1, where our approach will integrate experimental data with data-driven modeling to identify significant correlations and possible causal structures among the data elements, and with mechanistic modeling of cell-mediated immunity that translates biologically-based hypotheses into a dynamical system of time-dependent mathematical equations that can be used to simulate and test these hypotheses and to inform the design of subsequent experiments.</p>
<p>The proposed work plan begins with the collection and organization of quantitative and qualitative CSU generated experimental data that will then be used for data-driven and mechanistic modeling, with the analysis results and software modeling tools being made available through a web-based API. The milestones of this project are: (1) establishing protocols and standardized documentation for data collection and pre-processing from each CSU experimental type, (2) construction of the relational database (RDB) for CSU-generated experimental data, (3) collection of qualitative data describing key immune features as input for mechanistic modeling, (4) development of single-type data-driven analysis tools for each separate experimental system, (5) development of integrated data-driven analysis tools for the combined experimental data, (6) development of a dynamical systems model of cell-mediated immunity based on qualitative analysis results, (7) development of parameter estimation and model calibration procedures for the dynamical systems model, (8) development of software tools that provide for a start-to-finish process framework, and (9) development of an API for public access to relevant data and results. For each milestone, the gates for Go/No Go decisions will be based on the positive reproducibility of the major results by each of the individual CSU investigators. This approach will ensure the integration and quality control of each component within the entire framework.</p>
<p>Immunology data is collected from each CSU mouse experiment as both quantitative measurements and qualitative data that includes hypotheses regarding key biological constituents in the context of TB vaccine development. An RDB will be developed to provide access and queries to all combined data sets. Data-driven modeling will proceed directly from the quantitative data while mechanistic modeling will begin with the qualitative data, with the two modeling approaches increasingly informing each other as analyses proceed. The data-driven integrated data analysis will include visualization and statistical analyses and will also inform parameter estimation for the dynamical systems model. Software tools will be developed for all quantitative data and results, including user testing. A tailored user interface will provide access to all data and analysis results.</p>
<<<<<<< HEAD
<p>Methods. A series of three dedicated network database and computational data analysis servers will be configured for both internal (CSU) and remote access to experimental data and to the generated software. The first in this series will be a low cost, minimal architecture, development server for use during the base period. This development server will be then be scaled for intermediate development to establish resource requirements for an expected maximum workload, followed by a final production server running the completed workflow applications as the project deliverable. During the base funding period, we will be collecting sample datasets from each of the mouse TB infection experiments to catalog the data and metadata that are being collected. In subsequent funding periods, we will standardize the (meta)data collected from these groups and devise a database strategy to store these research outputs. This will include appropriate scaling for computer hardware and data storage options. Once the (meta)data is harmonized, we will construct the relational database. We intend to leverage open source database software like QPortal [5] and openBIS [6] which are designed to house this type of multi-omics data, to streamline this process. The final version of the web server will be housed centrally by Colorado State University’s Academic Computing and Networking Services division and maintained by qualified IT staff. These systems will also provide a web interface and an API interface that will allow users to query the data generated by this project, satisfying the data sharing plan. As data are ready to</p>
<p>be published, they will be deposited to the appropriate NCBI Database (such as Bioproject, biosample, and SRA) to increase discoverability. One possible unusual expense would occur in the event that the data stored locally is lost. In this case, we would incur egress fees from the cloud storage provider of 1-3 cents per gigabyte of data, depending on the speed of retrieval. The data-driven modeling will proceed during the base funding period to develop protocols for computationally reproducible data collection, pre-processing, analysis, and data-driven modeling of all data to be collected under the grant. These protocols will be developed using the rmarkdown [7] framework, to combine code with documentation describing all processing, analysis, and data-driven modeling choices. These protocols will be made publicly available and, once established, will be used throughout the project to ensure all experimental data are consistently and reproducibly processed and analyzed. These protocols will incorporate existing open source software tools, including xcms [8], ramclustR [9], flowCore [10], and openCyto [11]. Further, we will develop our own software tools to complement these existing tools for the purposes of our research, including novel tools for visualizing and integrating different types of data (e.g., metabolomics and flow cytometry). The final result of this data-driven modeling component will be data type-specific (e.g., identification of key metabolites) and integrative across data types (e.g., quantification of notable associations between specific metabolites and cell populations).</p>
<p>The mechanistic model component will be developed as a hierarchy of dynamical systems models of cell mediated immunity to TB infection with the basic approach described in previous models [2,3]. During the base funding period, qualitative data will be used to identify biologically important factors for the host immunity in each of the animal models under investigation. Common factors will be represented as graphs, and quantitative experimental data gaps needed to establish parameter estimates will be identified. Once appropriate experimental data is available, parameter estimation will be performed using Bayesian hierarchical modeling, and model simulations will be conducted using Monte Carlo methods to account for model uncertainty and population variability. Anticipated difficulties include additional gaps in experimental data that limit model identifiability, and hypotheses regarding the model elements that lead to model predictions that substantially disagree with corresponding experimental results. These issues will require additional iterations of model development, including possible new and targeted experimental studies. Schedule. The schedule for completion and delivery of items specified in the statement of work is shown in the table below, where arrows denote the duration and expected completion date.</p>
<p>References [1] Vodovotz Y, Xia A, Read EL, Bassaganya-Riera J, Hafler DA, Sontag E, Wang J, Tsang JS, Day JD, Kleinstein SH, Butte AJ, Altman MC, Hammond R, Sealfon SC. (2017) Solving Immunology? Trends Immunol. 38(2):116-127. [2] Friedman A, Turner J, Szomolay B. (2008) A model on the influence of age on immunity to infection with Mycobacterium tuberculosis. Exp Gerontol. 43(4):275-85. [3] Wigginton JE, Kirschner D. (2001) A model to predict cell-mediated immune regulatory mechanisms during human infection with Mycobacterium tuberculosis. J Immunol. 2001 Feb 1;166(3):1951-67. [4] Gideon HP, Skinner JA, Baldwin N, Flynn JL, Lin PL. (2016) Early Whole Blood Transcriptional Signatures Are Associated with Severity of Lung Inflammation in Cynomolgus Macaques with Mycobacterium tuberculosis Infection. [5] Mohr C, Friedrich A, Wojnar D, Kenar E, Polatkan AC, et al. (2018) qPortal: A platform for data-driven biomedical research. PLOS ONE 13(1): e0191603.<a href="https://doi.org/10.1371/journal.pone.0191603" class="uri">https://doi.org/10.1371/journal.pone.0191603</a> [6] Barillari C, Ottoz DSM, Fuentes-Serna JM, Ramakrishnan C, Rinn B, Rudolf F. (2016) openBIS ELN- LIMS: an open-source database for academic laboratories. Bioinformatics, 32(4): 638–640, <a href="https://doi.org/10.1093/bioinformatics/btv606" class="uri">https://doi.org/10.1093/bioinformatics/btv606</a> [7] JJ Allaire, Yihui Xie, Jonathan McPherson, Javier Luraschi, Kevin Ushey, Aron Atkins, Hadley Wickham, Joe Cheng and Winston Chang (2018). rmarkdown: Dynamic Documents for R. R package version 1.9. htt ps://CRAN.R-project.org/package=rmarkdown [8] Smith, C.A., Want, E.J., O’Maille, G., Abagyan,R., Siuzdak, G. (2006). XCMS: Processing mass spectrometry data for metabolite profiling using nonlinear peak alignment, matching and identification. Analytical Chemistry, 78, 779–787. [9] Broeckling, C.D., Afsar, F.A., Neumann, S., Ben-Hur, A., Prenni, J.E. (2014). RAMClust: a novel feature clustering method enables spectral-matching-based annotation for metabolomics data. Analytical Chemistry, 86(14), 6812-6817. [10] Ellis B, Haaland P, Hahne F, Le Meur N, Gopalakrishnan N, Spidlen J, Jiang M (2018). flowCore: flowCore: Basic structures for flow cytometry data. R package version 1.46.1. [11] Finak, Greg, Frelinger, Jacob, Jiang, Wenxin, Newell, Evan W., Ramey, John, Davis, Mark M., Kalams, Spyros A., De Rosa, Stephen C., Gottardo, Raphael (2014). “OpenCyto: An Open Source Infrastructure for Scalable, Robust, Reproducible, and Automated, End-to-End Flow Cytometry Data Analysis.” PLoS Computational Biology, 10(8), e1003806.</p>

</div>
=======
<p>You can label chapter and section titles using <code>{#label}</code> after them, e.g., we can reference Chapter <a href="intro.html#intro">1</a>. If you do not manually label them, there will be automatic labels anyway, e.g., Chapter <a href="#methods"><strong>??</strong></a>.</p>
<p>Figures and tables with captions will be placed in <code>figure</code> and <code>table</code> environments, respectively.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">par</span>(<span class="dt">mar =</span> <span class="kw">c</span>(<span class="dv">4</span>, <span class="dv">4</span>, .<span class="dv">1</span>, .<span class="dv">1</span>))
<span class="kw">plot</span>(pressure, <span class="dt">type =</span> <span class="st">&#39;b&#39;</span>, <span class="dt">pch =</span> <span class="dv">19</span>)</code></pre></div>
<div class="figure" style="text-align: center"><span id="fig:nice-fig"></span>
<img src="01-intro_files/figure-html/nice-fig-1.png" alt="Here is a nice figure!" width="80%" />
<p class="caption">
Figure 1.1: Here is a nice figure!
</p>
</div>
<p>Reference a figure by its code chunk label with the <code>fig:</code> prefix, e.g., see Figure <a href="intro.html#fig:nice-fig">1.1</a>. Similarly, you can reference tables generated from <code>knitr::kable()</code>, e.g., see Table <a href="intro.html#tab:nice-tab">1.1</a>.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">knitr<span class="op">::</span><span class="kw">kable</span>(
  <span class="kw">head</span>(iris, <span class="dv">20</span>), <span class="dt">caption =</span> <span class="st">&#39;Here is a nice table!&#39;</span>,
  <span class="dt">booktabs =</span> <span class="ot">TRUE</span>
)</code></pre></div>
<table>
<caption><span id="tab:nice-tab">Table 1.1: </span>Here is a nice table!</caption>
<thead>
<tr class="header">
<th align="right">Sepal.Length</th>
<th align="right">Sepal.Width</th>
<th align="right">Petal.Length</th>
<th align="right">Petal.Width</th>
<th align="left">Species</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">5.1</td>
<td align="right">3.5</td>
<td align="right">1.4</td>
<td align="right">0.2</td>
<td align="left">setosa</td>
</tr>
<tr class="even">
<td align="right">4.9</td>
<td align="right">3.0</td>
<td align="right">1.4</td>
<td align="right">0.2</td>
<td align="left">setosa</td>
</tr>
<tr class="odd">
<td align="right">4.7</td>
<td align="right">3.2</td>
<td align="right">1.3</td>
<td align="right">0.2</td>
<td align="left">setosa</td>
</tr>
<tr class="even">
<td align="right">4.6</td>
<td align="right">3.1</td>
<td align="right">1.5</td>
<td align="right">0.2</td>
<td align="left">setosa</td>
</tr>
<tr class="odd">
<td align="right">5.0</td>
<td align="right">3.6</td>
<td align="right">1.4</td>
<td align="right">0.2</td>
<td align="left">setosa</td>
</tr>
<tr class="even">
<td align="right">5.4</td>
<td align="right">3.9</td>
<td align="right">1.7</td>
<td align="right">0.4</td>
<td align="left">setosa</td>
</tr>
<tr class="odd">
<td align="right">4.6</td>
<td align="right">3.4</td>
<td align="right">1.4</td>
<td align="right">0.3</td>
<td align="left">setosa</td>
</tr>
<tr class="even">
<td align="right">5.0</td>
<td align="right">3.4</td>
<td align="right">1.5</td>
<td align="right">0.2</td>
<td align="left">setosa</td>
</tr>
<tr class="odd">
<td align="right">4.4</td>
<td align="right">2.9</td>
<td align="right">1.4</td>
<td align="right">0.2</td>
<td align="left">setosa</td>
</tr>
<tr class="even">
<td align="right">4.9</td>
<td align="right">3.1</td>
<td align="right">1.5</td>
<td align="right">0.1</td>
<td align="left">setosa</td>
</tr>
<tr class="odd">
<td align="right">5.4</td>
<td align="right">3.7</td>
<td align="right">1.5</td>
<td align="right">0.2</td>
<td align="left">setosa</td>
</tr>
<tr class="even">
<td align="right">4.8</td>
<td align="right">3.4</td>
<td align="right">1.6</td>
<td align="right">0.2</td>
<td align="left">setosa</td>
</tr>
<tr class="odd">
<td align="right">4.8</td>
<td align="right">3.0</td>
<td align="right">1.4</td>
<td align="right">0.1</td>
<td align="left">setosa</td>
</tr>
<tr class="even">
<td align="right">4.3</td>
<td align="right">3.0</td>
<td align="right">1.1</td>
<td align="right">0.1</td>
<td align="left">setosa</td>
</tr>
<tr class="odd">
<td align="right">5.8</td>
<td align="right">4.0</td>
<td align="right">1.2</td>
<td align="right">0.2</td>
<td align="left">setosa</td>
</tr>
<tr class="even">
<td align="right">5.7</td>
<td align="right">4.4</td>
<td align="right">1.5</td>
<td align="right">0.4</td>
<td align="left">setosa</td>
</tr>
<tr class="odd">
<td align="right">5.4</td>
<td align="right">3.9</td>
<td align="right">1.3</td>
<td align="right">0.4</td>
<td align="left">setosa</td>
</tr>
<tr class="even">
<td align="right">5.1</td>
<td align="right">3.5</td>
<td align="right">1.4</td>
<td align="right">0.3</td>
<td align="left">setosa</td>
</tr>
<tr class="odd">
<td align="right">5.7</td>
<td align="right">3.8</td>
<td align="right">1.7</td>
<td align="right">0.3</td>
<td align="left">setosa</td>
</tr>
<tr class="even">
<td align="right">5.1</td>
<td align="right">3.8</td>
<td align="right">1.5</td>
<td align="right">0.3</td>
<td align="left">setosa</td>
</tr>
</tbody>
</table>
<p>You can write citations, too. For example, we are using the <strong>bookdown</strong> package <span class="citation">(Xie <a href="#ref-R-bookdown">2019</a>)</span> in this sample book, which was built on top of R Markdown and <strong>knitr</strong> <span class="citation">(Xie <a href="#ref-xie2015">2015</a>)</span>.</p>

</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-xie2015">
<p>Xie, Yihui. 2015. <em>Dynamic Documents with R and Knitr</em>. 2nd ed. Boca Raton, Florida: Chapman; Hall/CRC. <a href="http://yihui.name/knitr/" class="uri">http://yihui.name/knitr/</a>.</p>
</div>
<div id="ref-R-bookdown">
<p>Xie, Yihui. 2019. <em>Bookdown: Authoring Books and Technical Documents with R Markdown</em>. <a href="https://CRAN.R-project.org/package=bookdown" class="uri">https://CRAN.R-project.org/package=bookdown</a>.</p>
</div>
</div>
>>>>>>> 377dd4d87328997d547aca3e4f4d2a05b33dc190
            </section>

          </div>
        </div>
      </div>

<a href="day-247.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/01-intro.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"download": ["impactb_book.pdf", "impactb_book.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
